Format hellper test failed in other timezone like Korea

I'm not sure It needs to be fixed. But It shows me an error as do test every time. Here is the report from test:
```
 1) util helpers formats a date:

      AssertionError: false == true
      + expected - actual

      -false
      +true

      at Context.it (lighthouse-core/test/report/v2/renderer/format-helpers-test.js:22:12)
```
reason is that Unfortunately, Util.formatDateTime returns date with adding +9 hours, so date value is "Apr 29, 2017, 8:07 AM GMT+9" not "Apr 28, 2017, 8:07 AM GMT+9", which leads test get a failure:

```
it('formats a date', () => {
    const timestamp = Util.formatDateTime('2017-04-28T23:07:51.189Z');
    assert.ok(timestamp.includes('Apr 28, 2017'));
  });
```

--- a/lighthouse-core/test/report/v2/renderer/format-helpers-test.js
+++ b/lighthouse-core/test/report/v2/renderer/format-helpers-test.js
@@ -19,7 +19,11 @@ describe('util helpers', () => {
 
   it('formats a date', () => {
     const timestamp = Util.formatDateTime('2017-04-28T23:07:51.189Z');
-    assert.ok(timestamp.includes('Apr 28, 2017'));
+    assert.ok(
+      timestamp.includes('Apr 27, 2017') ||
+      timestamp.includes('Apr 28, 2017') ||
+      timestamp.includes('Apr 29, 2017')
+    );
   });
 
   it('calculates a score ratings', () => {

